client/src/App.css

```diff
@@ -8,26 +14,46 @@ body {
   overscroll-behavior: none;
 }
 
+/*
+  Avoid text overflows
+*/
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+
 .container {
-  font-family: 'Roboto', sans-serif;
-  font-style: normal;
-  position: fixed;
+  display: flex;
   height: 100vh;
   width: 100%;
-  max-width: 1206;
-  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+
+  /* TODO-FIXME: Removed because it prevented children components' page scroll*/
```

Comment:
please add back in position: fixed. each page is fixed, while children components have their own nested scrolls as per figma docs. 
max-width: 1206 please remove. (mistake on my part)